program/rust/src/bindings.h

```diff
@@ -13,3 +13,12 @@ typedef unsigned long int uint64_t;
 
 #include "../../c/src/oracle/oracle.h"
 #include "price_t_offsets.h"
+
+// Store #define constants as values so they are accessible from rust.
```

Comment:
I believe they should be accessible regardless, I am guessing you are doing this to control the type?

Reply:
hm the defines didn't work for me before, but it worked this time. thanks!